Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs

```diff
@@ -38,7 +38,7 @@ public KryptonThemeBrowserForm(FormStartPosition? startPosition = FormStartPosit
 
             _formStartPosition = startPosition ?? FormStartPosition.CenterScreen;
 
-            _startIndex = startIndex ?? ThemeManager.GetThemeIndex();
+            _startIndex = startIndex ?? 0; // ThemeManager.GetThemeIndex();
```

Comment:
Why is `startIndex` a nullable item, just make it zero

Reply:
Need to figure out why the theme index numbers are broken

Reply:
Reverted changes